administrator/components/com_admin/script.php

```diff
@@ -8533,4 +8536,37 @@ protected function moveRemainingTemplateFiles()
 			}
 		}
 	}
+
+	/**
+	 * Ensure the core templates are correctly moved to the new mode.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function fixTemplateMode()
+	{
+		$db = Factory::getDbo();
+
+		array_map(function($template) use ($db)
+		{
+			$clientId = $template === 'atum' ? 1 : 0;
+			$query = $db->getQuery(true)
+				->update($db->quoteName('#__template_styles'))
+				->set($db->quoteName('inheritable') . ' = ' . $db->quote(1))
+				->where($db->quoteName('template') . ' = ' . $db->quote($template))
+				->where($db->quoteName('client_id') . ' = ' . $db->quote($clientId));
```

Comment:
```suggestion
				->where($db->quoteName('client_id') . ' = ' . $clientId);
```
Same here, it's an integer.

Reply:
doen